auth0/src/test/resources/credentials_openid.json

```diff
@@ -2,6 +2,5 @@
   "id_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL3NhbXBsZXMuYXV0aDAuY29tLyIsInN1YiI6ImF1dGgwfDUzYjk5NWY4YmNlNjhkOWZjOTAwMDk5YyIsImF1ZCI6Ikk5bWhVcmZrVEdGVldqbEVxWlNUQ0JVRkFCTGJKRkdMMyIsImV4cCI6MTQ2NTEwOTAzMywiaWF0IjoxNDY1MDczMDMzfQ.TdRc-lnVcX0LT7ZySzVysjVcYzAUIRnCPufTO8VV6g8",
   "access_token": "s6GS5FGJN2jfd4l6",
   "token_type": "bearer",
-  "expires_in": 86000,
-  "scope": "openid profile"
```

Comment:
removed the scope from here, since this is already a representation of the new Credentials object (only the required parameters are here). There's a second JSON file that also includes scope and refresh token, and is being referenced from tests already.